filebeat/beat/filebeat.go

```diff
@@ -80,7 +79,7 @@ func (fb *Filebeat) Run(b *beat.Beat) error {
 	}
 
 	// Start up spooler
-	go fb.spooler.Run()
+	fb.spooler.Start()
```

Comment:
Perhaps we should do the same for registrar (later), means having a start function.